pkg/kubelet/nodelease/controller_test.go

```diff
@@ -282,3 +282,111 @@ func TestRetryUpdateLease(t *testing.T) {
 		})
 	}
 }
+
+func TestUpdateUsingLatestLease(t *testing.T) {
+	node := &corev1.Node{
+		ObjectMeta: metav1.ObjectMeta{
+			Name: "foo",
+			UID:  types.UID("foo-uid"),
+		},
+	}
+
+	gr := schema.GroupResource{Group: "v1", Resource: "lease"}
```

Comment:
Group isn't actually v1.
```
import "k8s.io/kubernetes/pkg/apis/coordination"

notFoundErr := apierrors.NewNotFound(coordination.Resource("lease"), "foo")
```
coordination.Resource("lease")

Reply:
should be `coordinationv1`?

```go
import coordinationv1 "k8s.io/api/coordination/v1"

apierrors.NewNotFound(coordinationv1.Resource("lease"), "foo")
```

Reply:
in this context both are fine actually (because version is dropped for group resource)

Reply:
ok, thanks